pkg/scheduler/scheduler.go

```diff
@@ -106,6 +106,10 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		log := log.WithValues("queuedWorkload", klog.KObj(e.Obj), "clusterQueue", klog.KRef("", e.ClusterQueue))
 		if err := s.admit(ctrl.LoggerInto(ctx, log), &e); err != nil {
 			log.Error(err, "Failed admitting workload by clusterQueue")
+			if err := workload.UpdateWorkloadStatus(ctx, s.client, e.Obj, kueue.QueuedWorkloadAdmitted, corev1.ConditionFalse,
```

Comment:
nit:

```
err := call(parameters)
if err != nil {
```

Just because the line is long and the way you are splitting the parameters across lines is not very readable.